# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tinyxml/hotfix-stl.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Signed-off-by: Dennie Mahne <dennie.mahne@gmail.com>

--- tinyxml/tinyxml.h	2026-07-19 07:48:51.880774900 +0200
+++ tinyxml/tinyxml.h	2026-07-19 07:48:51.883810018 +0200
@@ -26,6 +26,8 @@
 #ifndef TINYXML_INCLUDED
 #define TINYXML_INCLUDED
 
+#define TIXML_USE_STL
+
 #ifdef _MSC_VER
 #pragma warning( push )
 #pragma warning( disable : 4530 )
